backend/sirius-web-diagrams-layout-api/src/main/java/org/eclipse/sirius/web/diagrams/layout/api/ILayoutService.java

```diff
@@ -26,6 +27,11 @@
 public interface ILayoutService {
     Diagram layout(IEditingContext editingContext, Diagram diagram);
 
+    /**
+     * Layout the specified node in the diagram.
+     */
+    Node layoutNode(IEditingContext editingContext, Diagram diagram, Node node);
```

Comment:
There's no need for such new public method. No one outside of the layout service is calling this method in your commit.